internal/uvm/create_lcow.go

```diff
@@ -305,6 +313,37 @@ func makeLCOWVMGSDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_
 		return nil, err
 	}
 
+	if opts.BundleDirectory == "" {
+		opts.BundleDirectory = os.TempDir()
```

Comment:
I'm not sure it's worth having this in here, since it would break immediately when we try to launch the second UVM, and the file paths conflict in the temp dir.

Reply:
yeah, that's fair.